framework/SupervisedLearning/ARMA.py

```diff
@@ -287,7 +338,7 @@ def __evaluateLocal__(self,featureVals):
       #debuggFile.writelines('signal_arma,'+','.join(str(x) for x in signal)+'\n')
 
       # Add fourier trends
-      if self.hasFourierSeries:
+      if target in self.fourierParams:
         signal += self.fourierResults[target]['predict']
         #debuggFile.writelines('signal_fourier,'+','.join(str(x) for x in self.fourierResults[target]['predict'])+'\n')
```

Comment:
remove?

Reply:
I'd rather not remove this, because I really want to output debugging signals to file, and I end up un-commenting these and using them a lot.  I would like to leave these here for the time when we are able to find the right way to output a SolutionExport from a RomTraining step.